redfish-core/lib/processor.hpp

```diff
@@ -900,6 +954,275 @@ inline void
     }
 }
 
+template <typename Handler>
+inline void getProcessorPaths(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
```

Comment:
Upstream style is to use the same variable name for commonly used variables. In this case, please use asyncResp instead of aResp.

Reply:
Will address this also as part of new commit (JIRA number will be attached)

Reply:
@deepakala-k 
It is better to rename `aResp` to `asyncResp` as a part of this PR, as it is not a logic or functional change.

Reply:
Sure